zubhub_frontend/zubhub/src/assets/js/styles/components/project/projectStyles.js

```diff
@@ -1,6 +1,12 @@
+/* eslint-disable no-dupe-keys */
 const styles = theme => ({
   root: {
-    maxWidth: 345,
+    [theme.breakpoints.down('md')]: {
```

Comment:
What does this do? Maybe I don't know. Just trying to learn. @NdibeRaymond probably can give us some insight

Reply:
@tuxology  okay well Material UI also has media query breakpoints just like bootstrap so --> .down('md') <-- means on screens from medium sizes to extra small it should apply that style to them. basically the margins were causing the horizontal overflow on medium devices downwards so i removed them on those devices with the them.breakpoints styling..

Reply:
@tuxology  Any screen size less than [900px](https://mui.com/customization/breakpoints/) will have 0 marginLeft and marginRight.

@Morai-C You don't need to specify units if the value is zero. It can be simply `marginLeft: 0`.

If you are setting margin left and right values you could also use `margin-inline`
https://developer.mozilla.org/en-US/docs/Web/CSS/margin-inline

Reply:
@kamthamc thanks for this.. i'm guessing the margin-inline would work the same way marginHorizontal on react native works?

Reply:
It should work in the same way. `margin-inline` is a standard css property. And I think `marginHorizontal` is specific to react-native